src/workers.ts

```diff
@@ -149,14 +149,13 @@ export async function createPenumbraWorker(
   const penumbraWorker: PenumbraWorker = {
     worker,
     id,
-    comlink: wrap(worker),
+    // eslint-disable-next-line no-spaced-func, func-call-spacing
+    comlink: wrap<new () => PenumbraWorkerAPI>(worker),
     busy: false,
   };
-  const Link = penumbraWorker.comlink;
-  const setup = new Link().then(async (thread: PenumbraWorkerAPI) => {
-    await thread.setup(id, proxy(reDispatchEvent));
-  });
-  await setup;
+  const RemoteAPI = penumbraWorker.comlink;
+  const remote = await new RemoteAPI();
+  await remote.setup(id, proxy(reDispatchEvent));
```

Comment:
nit: semantically Link makes more sense here since you instantiate comlinks

lgtm though as I see that this part is just upgrading to await

Reply:
Oh yeah happy to switch back. Only goal is to make it one name between them. Just picked this bc the type is Remote<MyWorkerInterface>